site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -226,6 +227,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.disableTLS`                       | Turns off TLS security for incoming connections to the allocator. | `false`                |
 | `agones.allocator.tolerations`                      | Allocator [toleration][toleration] labels for pod assignment                                    | `[]`                   |
 | `agones.allocator.affinity`                         | Allocator [affinity][affinity] settings for pod assignment                                      | `{}`                   |
+| `agones.allocator.annotations`                      | Allocator [annotations][annotations] settings for pod assignment                                | `{}`                   |
```

Comment:
Please also move this down to where we have the section "New Configuration Features:" - so they only get displayed on the next release :+1:

Reply:
Thank you. I fixed it with ab93fd6b5e51ebf65ce76e132a068b1e5584ff2e.